### DISCUSSION TITLE
Pretendard 개발 이야기를 11월 5일 Coloso에서 들어보세요.

### DISCUSSION THREAD

**Discussion by orioncactus** [orioncactus/pretendard#105] [Announcements]
## Pretendard 개발 이야기를 들어보세요.

![T/SCHOOL 2022](https://user-images.githubusercontent.com/7247848/196732209-4b3f365b-cc3d-42fa-8f85-33728ad725bc.gif)

안녕하세요! GitHub Discussions에서 처음으로 소식을 알려드립니다.

다가오는 11월 5일 토요일, [한국타이포그라피학회](http://k-s-t.org)와 [Coloso](http://coloso.co.kr), 그리고 [안그라픽스](http://agbook.co.kr)에서 주관하는, ‘디지털 타이포그래피’에 대한 통찰과 관점을 공유하는 행사인 [T/SCHOOL 2022](http://k-s-t.org/tschool)에서 Pretendard 개발 이야기를 주제로 강연을 진행하게 되었습니다.

지금 사전 신청을 통해 행사일에 Pretendard 개발 이야기를 포함한 다양한 디지털 타이포그래피 이야기를 무료 온라인 영상으로 들어보세요. 디지털 타이포그래피에 관심이 많으신 분들, 그리고 Pretendard가 어떻게 만들어졌고 어떻게 확장될 수 있는지 궁금하신 분들에게 이번 행사를 통해 수많은 영감을 줄 수 있길 기대합니다.

혹시 궁금한 부분이 있으시다면, Discussion은 언제든지 열려 있답니다! ٩(๑∂▿∂๑)۶♡

---

### [**Coloso에 수강 신청하러 가기**](https://coloso.co.kr/event/tschool2022)

**Comment by jsryudev** [comment]
orioncactus님 응원합니다!!
